build/packaging/post_build_script.sh

```diff
@@ -7,4 +7,5 @@
 
 set -eux
 
+rm -rf pip-out/
 echo "This script is run after building ExecuTorch binaries"
```

Comment:
Please turn this into a comment at the top of the script. Or if you leave it as an echo, it should be the first command that runs.